src/Admin/SiteHealth.php

```diff
@@ -335,29 +335,31 @@ static function ( $available_service ) {
 	 * @return array
 	 */
 	public function page_cache() {
-		$has_page_caching = $this->has_page_caching();
+		$page_cache_status = $this->get_page_cache_status();
 
-		$badge_color = 'orange';
-		$status      = 'recommended';
+		$badge_color = 'red';
+		$status      = is_wp_error( $page_cache_status ) ? 'critical' : $page_cache_status;
 		$label       = __( 'Page caching is not detected', 'amp' );
```

Comment:
This may be more accurate to say the page caching is not detected _and_ the response time is slow. Otherwise, the case of page caching being detected and yet the response time being OK is captured below.